### PR TITLE
Add "single" option to RuntimeChunkPlugin

### PR DESCRIPTION
**Pull request by benthemonkey**
**What kind of change does this PR introduce?**

Add config option `optimization.runtimeChunk: 'single'` to generate a single runtime chunk for all entrypoints instead of each entrypoint getting their own runtime.

**Did you add tests for your changes?**

Basic tests for the existence of runtime files, nothing that tests that the contents of the runtime file are generated correctly.

**If relevant, link to documentation update:**

**Summary**

My company's webpack setup involves including multiple entrypoint files on a page, so adding a separate runtime chunk for each entrypoint results in too many parallel loading chunks.

See @sokra's comment in the alpha.5 issue: https://github.com/webpack/webpack/issues/6357#issuecomment-360435877

**Does this PR introduce a breaking change?**

No, just a new option in the config and the RuntimeChunkPlugin.

**Other information**

Let me know if you approve of the way I am handling passing in the option. Once I get a 👍 I'll update docs and add tests. I mostly wanted to make sure you won't want to allow a custom name for the runtime chunk.

### CODE REVIEW

File: schemas/WebpackOptions.json [hunk 1]
```diff
@@ -1484,7 +1484,16 @@
...
         "runtimeChunk": {
           "description": "Create an additional chunk which contains only the webpack runtime and chunk hash maps",
-          "type": "boolean"
+          "oneOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "enum": [
+                "single"
```
Comment (ooflorent) [inline schemas/WebpackOptions.json]: Please also add the counterpart

Comment (benthemonkey) [reply schemas/WebpackOptions.json]: Do you mean I should add another string option called `multiple`?

Comment (ooflorent) [reply schemas/WebpackOptions.json]: Yes.

**Comment by sokra** [comment]
Thanks

**Pull request merged**